frontend/src/app/home/components/map/clickables/clickable-others/clickable-others.component.ts

```diff
@@ -21,6 +22,27 @@ export class ClickableOthersComponent implements OnChanges {
     }
   }
 
+  whatElement(event: Event){
+
+    this.mainService.all.forEach(element => {
+      if (element.numberSVG === event.srcElement.id ) {
+        this.detailArray = element.map();
+       if(element.numberSVG[0] == 'k'){
```

Comment:
Jeżeli będziemy bazować na evencie a nie na elementcie to możemy łatwo poznać rodzica, ale ten przypadek pokazuje, że o wiele lepiej zrobić metode na modelu do zwrócenia tekstu tooltipa